src/main/java/org/wildfly/security/sasl/gs2/Gs2.java

```diff
@@ -42,26 +43,10 @@
     // Non-OID-derived mechanism names
     public static final String GS2_KRB5 = "GS2-KRB5";
     public static final String GS2_KRB5_PLUS = "GS2-KRB5-PLUS";
-    public static final Oid OID_KRB5;
```

Comment:
This one on the other hand is not recorded as public API so we can probably get away with this one.